client/src/app/components/FilterToolbar/MultiselectFilterControl.tsx

```diff
@@ -138,8 +138,6 @@ export const MultiselectFilterControl = <TItem,>({
     const input = textInput?.toLowerCase();
 
     return renderSelectOptions((optionProps, groupName) => {
-      if (!input) return false;
```

Comment:
Needed to remove this to fix the select options not rendering on first render after converting the app name filter to multiselect for the app inventory table.